pkg/test/dualstack/cloud_providers.go

```diff
@@ -397,6 +397,35 @@ func (a equinix) CloudSpec() models.CloudSpec {
 	}
 }
 
+type vsphere struct{}
+
+var _ clusterSpec = vsphere{}
+
+func (a vsphere) NodeSpec() models.NodeCloudSpec {
+	return models.NodeCloudSpec{
+		Vsphere: &models.VSphereNodeSpec{
+			Template:   "ubuntu-20.04",
+			CPUs:       2,
+			Memory:     4096,
+			DiskSizeGB: 10,
+		},
+	}
+}
+
+func (a vsphere) CloudSpec() models.CloudSpec {
+	return models.CloudSpec{
+		DatacenterName: "vsphere-ger",
+		Vsphere: &models.VSphereCloudSpec{
+			Username: "VSPHERE_USERNAME",
```

Comment:
did you mean `os.Getenv("VSPHERE_USERNAME")`?

Reply:
OFC :) thanks